packages/tooltip/src/components/tooltip/WithTooltip.md

```diff
@@ -0,0 +1,59 @@
+# WithTooltip
+
+`WithTooltip` can add a tooltip to any component.
+
+```
+<WithTooltip
+  placement={"bottom"}
+  content={<StandardText>Nice content on bottom</StandardText>}
+>
+  <StandardText>Hover me</StandardText>
+</WithTooltip>
+```
+
+This uses popper.js to position the tooltip.
+
+## `placement` prop
+
+Specifies which side of `children` the tooltip should be positioned.
+This is optional. Can be `"bottom"`, `"top"`, `"left"` or `"right"`.
```

Comment:
What is default?

Reply:
Default is undefined and falls back to Popper default behavior.